.github/label-actions.yml

```
@@ -217,15 +217,57 @@
     Thanks, the Renovate team
   close: true
   close-reason: 'not planned'
-  lock: true
-  lock-reason: 'resolved'
 
-'format-code-comments':
+'auto:format-code':
   comment: >
     Hi, please format any copy/pasted code or logs so they're readable.
 
 
     You can find a Markdown code formatting guide [here](https://www.markdownguide.org/basic-syntax/#code) as well as some GitHub-specific information formatting code blocks [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).
 
 
+    Thanks, the Renovate team
+
+'auto:discussion-closed':
+  comment: >
+    Hi there,
+
+
+    This discusion is being closed because it's missing some details, or something else about it has made it difficult or impossible for us to help resolve the issue.
+
+
+    We choose to close and later delete such discussions in order to keep a high quality of discussions in this repository for users who search for answers.
```

```suggestion
    We want high quality discussions, so users can search them for answers.
    This means we'll close (and later delete) low quality discussions.
```

We should split the sentence into two parts:

- Why we want high-quality discussions
- What this means for the current discussion

I'll let you decide on the tone/voice of this message. 😉